packages/pass-style/test/passStyleOf.test.js

```diff
@@ -13,7 +13,7 @@ const harden = /** @type {import('ses').Harden & { isFake?: boolean }} */ (
   global.harden
 );
 
-const { getPrototypeOf, defineProperty } = Object;
+const { getPrototypeOf, defineProperty, freeze } = Object;
```

Comment:
I get dizzy with all the newly-introduced TODOs.  Can we instead just alias `suppressTrapping` to `harden` once per module?  If you want to leave a TODO, maybe write it once per module in that place?

Here's a suggestion, but I am not attached to how you actually code this if you like the idea behind it:

```suggestion
const { getPrototypeOf, defineProperty, freeze, suppressTrapping = harden } = Object;
```

Reply:
Much better, thanks!

Done